packages/test/test-end-to-end-tests/package.json

```diff
@@ -134,6 +134,9 @@
 		"@types/uuid": "^8.3.0",
 		"concurrently": "^7.6.0",
 		"eslint": "~8.6.0",
+		"mocha-json-output-reporter": "^2.0.1",
+		"mocha-multi-reporters": "^1.5.1",
+		"moment": "^2.21.0",
```

Comment:
I know this is in the original change, but I wonder why this is needed.

Reply:
I was under the impression that it was a peer of one of the other two, but I will confirm.

Reply:
Yes, it is a peer dep of `mocha-json-output-reporter`.